docs/source/set_dev_env.rst

```diff
@@ -41,19 +41,26 @@ Create database and user::
 For convenience for development, user, password and database name are already
 set in `tcms/settings/devel.py` with default value. Each of them is `nirrate`.
 
-Load database schema::
+.. note::
 
-    mysql -uroot nitrate < contrib/sql/nitrate_db_setup.sql
+    You may want to adjust the database and/or other configuration settings.
+    Override them in ``./tcms/settings/devel.py`` if necessary. While the
+    Nitrate team prefers MySQL, sqlite has been verified to work for development
+    and some people have reported using PostgreSQL in production!
```

Comment:
I don't know anyone run Nitrate with PostgreSQL. Actually, there are some code, which is raw SQL statements, only specific to MySQL in ``report`` app.